study.md

```diff
@@ -111,15 +123,16 @@ The array should contain two strings, "drinking beer" and "propane and propane
 accessories".
 
 ```javascript
-// your answer here
+hank.middle-name = "Rutherford"
```

Comment:
Special characters in key names need to be wrapped in quotes and brackets like so: `hank['middle-name']`.